docs/docs/pipeline.md

```diff
@@ -384,6 +384,28 @@ when:
   instance: stage.drone.company.com
 ```
 
+Execute a step only on commit with certain files added/removed/modified:
+
+**NOTE: Feature is only available for Github repositories.**
+
+```diff
+when:
+  path: src/*
```

Comment:
Can you make sure the syntax in this doc is up to date?

I tried both from this docs, and neither of them worked :thinking:

Reply:
let me get my local setup working again, and will provide a working example. I haven't touched it in a while 😓

Reply:
```suggestion
  path: "src/*"
```
the wildcard must be escaped in YAML